project/client/js/InventoryHelper.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Created by vova on 06.10.2015.
+ */
+Template.inventory.helpers({
+    problem: function(){
```

Comment:
problems is an array, that I use in code. Problem - is an array, that I give in template.